src/OpenTracingMock/Tracer.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace OpenTracingMock;
+
+use OpenTracing\Exceptions\UnsupportedFormat;
+use OpenTracing\SpanOptions;
+use OpenTracing\Tracer as OTTracer;
+use OpenTracing\SpanContext as OTSpanContext;
+
+final class Tracer implements OTTracer
+{
+    /**
+     * @var array|Span[]
+     */
+    private $spans = [];
+
+    /**
+     * @var array|callable[]
+     */
+    private $injectors;
+
+    /**
+     * @var array|callable[]
+     */
+    private $extractors;
+
+    private function __construct(array $injectors, array $extractors)
+    {
+        $this->injectors = $injectors;
+        $this->extractors = $extractors;
+    }
+
+    public static function create(array $injectors = [], array $extractors = [])
+    {
+        return new self($injectors, $extractors);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function startSpan($operationName, $options = [])
+    {
+        if (!($options instanceof SpanOptions)) {
+            $options = SpanOptions::create($options);
+        }
+
+        if ($options->getReferences()) {
```

Comment:
I think the condition should be inverted.

Reply:
Could an empty check be an alternative?
```if (empty($options->getReferences())) { ... ```